core/templates/dev/head/filters/filtersSpec.js

```diff
@@ -336,7 +336,7 @@ describe('Testing filters', function() {
           x: 2
         }
       };
-      var interactionIdMath = 'TextInput';
+      var interactionIdMath = 'MultipleChoiceInput';
```

Comment:
Here and below, the name of the variable does not seem to match the interaction ID. Not sure if that's intended, but figured I'd point it out.

Reply:
This interaction uses latex for choices, so it's named as `interactionIdMath` whereas around line 369 we have used mixed tags in the choice (image tag, link etc.)so it's named as `interactionIdMixed`. I'll try adding a better name.

Reply:
interactionIdWithMathChoices perhaps.